modules/health_quest/app/services/health_quest/resource/client_model/questionnaire_response.rb

```diff
@@ -65,8 +65,7 @@ def set_source
         end
 
         def set_questionnaire
-          questionnaire_id = data.dig(:questionnaire, :id) || DEFAULT_QUESTIONNAIRE_ID
-
+          questionnaire_id = data.dig(:questionnaire, :id)
```

Comment:
Do you want an || statement here still?

Reply:
@LindseySaari we don't want the default to be set for the title. If it's missing we actually want to see that it's missing as opposed to giving the user and the system the wrong idea of what the `questionnaire` actually is.

Reply:
Ok, great! 👍